### PR TITLE
Remove sass styles and packages

### LINKED ISSUE #774: Remove Vocabulary CSS styles and create issues to fix areas that need fixing before it can be removed in production
## Problem

We currently ship a large amount of unused CSS due to the inclusion of Vocabulary CSS library. With the redesign, most of the website has been re-written with Tailwind and no longer depends on Vocabulary. We can start a milestone to track the progression of re-writing the final areas of the site that still use Vocabulary CSS into Tailwind and put issues describing each page/area that needs to be re-written.

## Description

Remove Vocabulary CSS from your local dev site and document what breaks. Create discrete issues for each page/area of the site that breaks and needs to be re-written. Put them in a new "Remove Vocabulary CSS" milestone.

To be clear, the deliverable for this issue is a milestone with other issues, not a PR that actually fixes these issues or removes Vocabulary.

## Additional context

This could cut down our CSS by an estimated 75% or more.

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #774 by @sarayourfriend
~~Blocked by #1317~~

## Description

This PR removes all the legacy style files that used Sass. It also removes `sass` and `sass-loader` from dependencies.
The Sources page was simplified a little bit, too.

I specifically added you to the reviewers list, @panchovm , because you can see style regressions much faster than anyone else :) 

## Testing Instructions

Open the app. There should be no drastic changes in the spacing, font sizes, button styles. Any inconsistencies with the mockups can be fixed in later PRs.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -139,8 +139,6 @@
     "postcss": "^8.4.12",
     "prettier": "^2.2.1",
     "rimraf": "^3.0.2",
-    "sass": "^1.49.9",
-    "sass-loader": "^10.2.1",
```
Comment (sarayourfriend) [inline package.json:143]: :tada: Yay!!!

**Review by sarayourfriend — CHANGES_REQUESTED** [review]
LGTM, just a couple things.

Can we also remove sass/scss entries from the following places, for consistency?

https://github.com/WordPress/openverse-frontend/blob/d6775aee4b921d8d3d158d5b3a213f81341bbe4f/nuxt.config.ts#L182

https://github.com/WordPress/openverse-frontend/blob/df16c251e8b543fd49af292f7067b7ac4797c3c5/jest.config.js#L21

https://github.com/WordPress/openverse-frontend/blob/a92edbf53e48c7733d74c6ce87a570cbc80b615a/.github/workflows/bundle_size.yml#L10

Also, the extension page layout is very broken now.
image

Other than that I couldn't find any issues :tada: 

It does look like some of the layout shifted on the sources page and some other pages, but only very slightly. I agree they are minor and can be addressed later if there are discrepancies.

**Comment by obulat** [comment]
> LGTM, just a couple things.
> 
> Can we also remove sass/scss entries from the following places, for consistency?
> 
> https://github.com/WordPress/openverse-frontend/blob/d6775aee4b921d8d3d158d5b3a213f81341bbe4f/nuxt.config.ts#L182
> 
> https://github.com/WordPress/openverse-frontend/blob/df16c251e8b543fd49af292f7067b7ac4797c3c5/jest.config.js#L21
> 
> https://github.com/WordPress/openverse-frontend/blob/a92edbf53e48c7733d74c6ce87a570cbc80b615a/.github/workflows/bundle_size.yml#L10
> 
> Also, the extension page layout is very broken now. image
> 
> Other than that I couldn't find any issues 🎉
> 
> It does look like some of the layout shifted on the sources page and some other pages, but only very slightly. I agree they are minor and can be addressed later if there are discrepancies.

Ah, thank you for catching this! I did **_not_** look at the extension page.

**Comment by sarayourfriend** [comment]
It'd be nice to have simple snapshot tests for each of the content pages, like the sources ones. Though now that you've removed the global styles I'm much less worried about them suddenly changing without us noticing!

File: src/components/VCheckbox/meta/VCheckbox.stories.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ const LicenseCheckboxStory = (args, { argTypes }) => ({
   props: Object.keys(argTypes),
   methods: {
     onChange(params) {
-      window.alert('License checkbox checked! ' + JSON.stringify(params))
+      window.alert('License checkbox checked. ' + JSON.stringify(params))
```
Comment (obulat) [inline src/components/VCheckbox/meta/VCheckbox.stories.js:49]:
```suggestion
      window.alert('License checkbox checked! ' + JSON.stringify(params))
```

**Review by panchovm — APPROVED** [review]
Everything looks great to me. Nice work ✨

**Comment by obulat** [comment]
Wow, I did not expect such an easy approval from you, @panchovm! Thank you!

**Comment by panchovm** [comment]
I went through comparing the mockups and seeing if nothing is broken 😃

**Comment by obulat** [comment]
@sarayourfriend, I've added the snapshots for content pages. I wonder if I should click on the translation bar 'Close' button for 'rtl' snapshots. The translation banner normally should not change, but if it does, there will be so many snapshots to update!

**Comment by sarayourfriend** [comment]
@obulat That probably does make sense, you're right. As long as we have tests specifically targeting the translation banner then I think it's a good idea.

Once we have the UI state cookie in place we can simplify that as well by not having to click "close" on every test, we can just set the dismissed cookie at the top of the describe block.

**Review by sarayourfriend — APPROVED** [review]
LGTM! I had a weird thing happening where I wasn't able to see the back to top button but I am not able to reproduce it :shrug: Great work! This is such an exciting development.

**Pull request merged**